WikiWatershed/model-my-watershed
# Add Celery task for analyze and remove analyze app

This PR removes the short-lived analyze Django app and creates a task for it in the jobs app. The front-end is updated to handle these changes.

See commit messages for details.

To test:
- Select a feature from the draw menu. The analyze window should open. The header should contain some information, but the main area should be blank. In about 3 seconds, it will get filled in with the table and chart.

Display and loading of error messages will be handled in #80.
